build.gradle

```diff
@@ -139,35 +137,6 @@ subprojects {
 
   apply plugin: 'java-library'
 
-  if (project.name in REVAPI_PROJECTS) {
-    apply plugin: 'com.palantir.revapi'
-    revapi {
-      oldGroup = project.group
-      oldName = project.name
-      oldVersion = "1.5.0"
-    }
-
-    tasks.register('showDeprecationRulesOnRevApiFailure') {
-      doLast {
-        throw new RuntimeException("==================================================================================" +
-                "\nAPI/ABI breaks detected.\n" +
```

Comment:
Can we still add this output when RevAPI finds breaking changes? I believe this output is quite helpful and we mention it in the docs

Reply:
Sure, the script can display this message. Currently, it displays "API breaks detected check revapi.out for details".

Reply:
I added the same message and I will update website (contributing section) how to add accepted breaks.

Reply:
@Fokko what do you mean by "pre-create" the file ?

Reply:
For the "accepted breaking changes", I added an example file: `dev/.revapi.json` which is the standard revapi configuration file (the `revapi.yml` is gradle plugin specific).

Reply:
That's exactly what I ment :D